.github/workflows/release.yml

```diff
@@ -19,15 +19,18 @@ jobs:
     name: Run Integration Tests
     runs-on: ubuntu-latest
     steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v5
+
+      - name: Check Go Versions
+        run: ./scripts/check_go_version.sh
+
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version: "1.22"
+          go-version: "1.24.2"
```

Comment:
Why do these need this to be in sync? IIRC Go will run as the version defined in `go.mod` even if it's a lower version. Higher version of course takes precedence.

See:

```
~
❯ go1.23.7 version
go version go1.23.7 darwin/arm64
```

```
~/Code/coder main*
❯ go1.23.7 version
go version go1.24.6 darwin/arm64
```

Btw, `go.mod` says `1.24.6`.

Reply:
Ignore my comment about 1.24.6 btw, originally thought this was a PR to coder/coder repo 😄